app/helpers/template_helpers.py

```diff
@@ -250,23 +243,15 @@ def get_static_content_urls(language_code: str, base_url: str, schema_theme: str
         )
 
     elif schema_theme == "census-nisra":
-        help_path = (
-            "help/help-with-the-questions/online-questions-help/"
-            if schema_theme == "census-nisra"
-            else "help/how-to-answer-questions/online-questions-help/"
-        )
+        help_path = "help/how-to-answer-questions/online-questions-help/"
```

Comment:
This should be `"help/help-with-the-questions/online-questions-help/"`?

Reply:
Too many PRs today! Thanks!